include/kframework/evm-types.md

```diff
@@ -257,10 +257,10 @@ A cons-list is used for the EVM wordstack.
 ```k
     syntax Bytes ::= #take ( Int , Bytes ) [klabel(takeBytes), function, total]
  // ---------------------------------------------------------------------------
-    rule #take(N, _BS:Bytes) => .Bytes                                      requires                                        notBool N >Int 0
-    rule #take(N,  BS:Bytes) => #padRightToWidth(N, .Bytes)                 requires notBool lengthBytes(BS) >Int 0 andBool         N >Int 0
-    rule #take(N,  BS:Bytes) => BS ++ #take(N -Int lengthBytes(BS), .Bytes) requires         lengthBytes(BS) >Int 0 andBool notBool N >Int lengthBytes(BS)
-    rule #take(N,  BS:Bytes) => BS [ 0 .. N ]                               requires         lengthBytes(BS) >Int 0 andBool         N >Int lengthBytes(BS)
+    rule #take(N, _BS:Bytes) => .Bytes                                          requires                                        notBool N >Int 0
+    rule #take(N,  BS:Bytes) => #padRightToWidth(N, .Bytes)                     requires notBool lengthBytes(BS) >Int 0 andBool         N >Int 0
+    rule #take(N,  BS:Bytes) => BS +Bytes #take(N -Int lengthBytes(BS), .Bytes) requires         lengthBytes(BS) >Int 0 andBool notBool N >Int lengthBytes(BS)
+    rule #take(N,  BS:Bytes) => #range(BS, 0, N)                                requires         lengthBytes(BS) >Int 0 andBool         N >Int lengthBytes(BS)
```

Comment:
I believe that this `#take` is actually no longer used. There is another `#take` for WordStack that is still used, but this one is not I believe. I'm pretty sure the same is true for `#drop` over `Bytes` type, no longer used. There is another `#drop` over wordstack that is still used, that one should stay.

Reply:
Done.